pkg/kubelet/cm/topologymanager/policy_single_numa_node.go

```diff
@@ -49,10 +49,10 @@ func filterSingleNumaHints(allResourcesHints [][]TopologyHint) [][]TopologyHint
 	for _, oneResourceHints := range allResourcesHints {
```

Comment:
I think you should remove the package in https://github.com/kubernetes/kubernetes/blob/master/hack/.staticcheck_failures

Reply:
I didn't see `kubelet` related pkg there

Reply:
Maybe, the `.staticcheck_failures` needs an update. @BenTheElder @nolancon @sjenning

Reply:
this file is automatically kept up to date, PRs won't merge if it's up to date.

are you running staticcheck using the rule in the repo or with your own install? we use a specific version in CI